ggml/src/ggml-vulkan/vulkan-shaders/quantize_q8_1.comp

```diff
@@ -45,11 +45,7 @@ void quantize() {
     const uint ib = wgid * blocks_per_group + block_in_wg;
     const uint iqs = tid % 8;
 
-#ifndef QBLOCK_X4
-    if (ib >= gl_NumWorkGroups.x * blocks_per_group) {
```

Comment:
In the original code, I couldn't tell how this would ever return early so I just deleted it. If I've misunderstood, let me know. Or we could remove this path entirely since it's unused.